cases/cabauw/cabauw_rt_input.py

```diff
@@ -93,16 +93,19 @@ def generate_case(
         copy('{}/misc/van_genuchten_parameters.nc'.format(microhh_path), 'van_genuchten_parameters.nc')
 
         if gpt_set == '256_224':
-            copy('{}/rte-rrtmgp-cpp/rte-rrtmgp/rrtmgp/data/rrtmgp-data-lw-g256-2018-12-04.nc'.format(microhh_path), 'coefficients_lw.nc')
```

Comment:
Has this location changed? If so, we should update the relevant path in `microhh_tools.py` as well